frontend/src/components/ViewTrips/index.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
-import app from '../Firebase';
-import TripsContainer from './trips-container.js'
+
+import Button from 'react-bootstrap/Button';
+
+import app from '../Firebase/index.js';
+import Header from '../Header/index.js';
+import AddTrip from '../AddTrip/index.js'
+import TripsContainer from './trips-container.js';
```

Comment:
I believe you can do:

```
import app from '../Firebase';
import Header from '../Header';
...
```
instead of importing index.js.

Reply:
Yeah I had it as this originally but Hiram mentioned this in Keiffer's PR and I am not sure if this applies to this case too since I am technically importing a file:

> According to the style guide, you must include the ".js" suffix when importing files: https://google.github.io/styleguide/jsguide.html#es-module-imports

thoughts?

Reply:
I think we agreed that it applied to non-index.js functions.

Reply:
Gotcha, thanks!